src/wallet/wallet2.cpp

```diff
@@ -779,16 +809,15 @@ bool wallet2::init(std::string daemon_address, boost::optional<epee::net_utils::
   return m_http_client.set_server(get_daemon_address(), get_daemon_login(), ssl);
 }
 //----------------------------------------------------------------------------------------------------
-bool wallet2::is_deterministic() const
+bool wallet2::is_deterministic()
```

Comment:
This and `get_seed` can both remain `const` as no encryption is happening inside.